packages/file-menu/package.json

```diff
@@ -0,0 +1,44 @@
+{
+    "name": "@dhis2/d2-ui-file-menu",
+    "version": "1.0.0",
+    "description": "File menu component for DHIS2",
+    "main": "./build/cjs/index.js",
+    "module": "./build/es/index.js",
+    "files": [
+        "build"
+    ],
+    "license": "BSD-3-Clause",
+    "author": "Edoardo Sabadelli <edoardo@dhis2.org>",
+    "peerDependencies": {
+        "d2": "^29.1.1",
+        "react": "^15.0.0 || ^16.0.0",
+        "react-dom": "^15.0.0 || ^16.0.0"
+    },
+    "scripts": {
```

Comment:
Don't we need the string extraction scripts and deps here?

https://github.com/dhis2/d2-i18n#in-apps-packagejson

Reply:
Added deps and scripts and generated the pot file.